### ISSUE TITLE
Consumer' object is not iterable

### ISSUE THREAD

**Issue by jmcgrath207** [hampsterx/async-kinesis#14]
I get this when I try to use the example for consumers in the README.md

```

    async with Consumer(stream_name="test_stream", endpoint_url=aws_endpoint) as consumer:
        while True:
            async for item in consumer:
                print(item)

```

ERROR
```
 for item in consumer:
 TypeError: 'Consumer' object is not iterable
```

**Comment by jmcgrath207** [comment]
Found the issue. This is cased by a breaking change in `aiobotocore==1.0.0` . When I rolled back to `aiobotocore==0.12.0` everything worked. This is due to https://github.com/aio-libs/aiobotocore/releases/tag/1.0.0.

**Comment by hampsterx** [comment]
#15 fixed with 0.5.0 release thnx~
